### PR TITLE
Browser comp

### PR DESCRIPTION
**Pull request by kyam-mayden**
Browser comp issues resolved

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -108,9 +116,9 @@ <h3>
 			</article>
 		</div>
 	</main>
-	<footer class="container" id="foot">
+	<div class="container" id="foot">
```
Comment (karimkefi) [inline index.html]: change back to footer

File: styles.css [hunk 2]
```diff
@@ -293,13 +293,27 @@ article{
...
 }
 
+.clearfix {
+	display: block;
+	height: 1%;
+}
+
+.clearfix:after {
+	content: ".";
+	display: block;
```
Comment (karimkefi) [inline styles.css]: do you have to repeat display block if the parent already has it?

File: styles.css [hunk 3]
```diff
@@ -293,13 +293,27 @@ article{
 	font-weight:700;
 }
 
+.clearfix {
+	display: block;
+	height: 1%;
+}
+
+.clearfix:after {
+	content: ".";
```
Comment (karimkefi) [inline styles.css]: is this full stop adding anything?

File: index.html (hunk 1 above)
Comment (kyam-mayden) [reply index.html]: Done

File: styles.css (hunk 3 above)
Comment (kyam-mayden) [reply styles.css]: resolved.

File: styles.css (hunk 2 above)
Comment (kyam-mayden) [reply styles.css]: Resolved.

File: index.html [hunk 4]
```diff
@@ -116,9 +116,9 @@ <h3>
 			</article>
 		</div>
 	</main>
-	<div class="container" id="foot">
+	<footer class="container" id="foot">
```
Comment (nsantiagoblair) [inline index.html]: can this `id` be a `class`

**Review by nsantiagoblair — COMMENTED** [review]
🎉 🌮

File: index.html (hunk 4 above)
Comment (kyam-mayden) [reply index.html]: ID is needed for presedence

**Review by nsantiagoblair — APPROVED** [review]
AWESOME!!!

**Pull request merged**